.travis.yml

```diff
@@ -1,29 +1,23 @@
 language: c
 env:
+###############################################################################
   # FIXME: If we want to check assembly too SYNTAX_ONLY needs to be 0
   # FIXME: We'd like to build with WARNINGS_AS_ERRORS=1 but there are too
   # many warnings firing right now
   # Note: Can't use CC variable as TravisCI will override it
-  - C_COMPILER=gcc-4.8 SYNTAX_ONLY=1 WARNINGS_AS_ERRORS=0 VERBOSE=0 SUPPRESS_WARNINGS=1 REPORT_CC_FILE=2
+  - C_COMPILER=gcc SYNTAX_ONLY=1 WARNINGS_AS_ERRORS=0 VERBOSE=0 SUPPRESS_WARNINGS=1 REPORT_CC_FILE=2
```

Comment:
@PhilippWendler ``c/Makefile-README.md`` already heavily documents how to use the build system.

The issue here is that the build system failed automatically determine what your compiler is. Take a look at these lines in ``Makefile.rules``.

```
CC_VERSION_INFO :=$(shell $(CC) --version)
CC_IS_GCC:=$(shell echo '$(CC_VERSION_INFO)' | $(GREP) -ic gcc)
CC_IS_CLANG:=$(shell echo '$(CC_VERSION_INFO)' | $(GREP) -ic clang)
```

these determine the compiler and something is going wrong in your environment.